translations/ru.po

```diff
@@ -84,63 +84,62 @@ msgid "Insert After"
 msgstr "Вставить после"
 
 #: src/config_classes/Formatter.gd
-#, fuzzy
 msgid "Always"
-msgstr "Всегда активно."
+msgstr "Всегда активно"
```

Comment:
It shouldn't be "always active". That string is retired, this is in the context of "Always / Sometimes / Never" sorts of dropdowns.

Reply:
I guess I just missed string haha